Orange/widgets/widget.py

```diff
@@ -251,22 +255,27 @@ def createPixmapWidget(self, parent, iconName):
         self.layout().addWidget(splitter)
 
         if self.want_control_area:
-            self.controlArea = gui.widgetBox(splitter,
-                                             orientation="vertical",
-                                             margin=0)
+            left_side = gui.vBox(splitter, spacing=0)
             splitter.setSizes([1])  # Results in smallest size allowed by policy
 
-            if self.graph_name is not None or hasattr(self, "send_report"):
-                leftSide = self.controlArea
-                self.controlArea = gui.widgetBox(leftSide, margin=0)
-            if self.graph_name is not None:
-                self.graphButton = gui.button(leftSide, None, "&Save Graph")
-                self.graphButton.clicked.connect(self.save_graph)
-                self.graphButton.setAutoDefault(0)
-            if hasattr(self, "send_report"):
-                self.report_button = gui.button(leftSide, None, "&Report",
-                                                callback=self.show_report)
-                self.report_button.setAutoDefault(0)
+            if self.want_standard_buttons_box:
+                self.controlArea = gui.vBox(
+                    left_side, addSpace=0)
+                self.standardButtons = gui.widgetBox(
+                    left_side, addSpace=0, spacing=9,
+                    orientation=self.want_standard_buttons_box)
+                if self.graph_name is not None:
+                    self.graphButton = gui.button(
+                        self.standardButtons, None, "&Save Graph",
```

Comment:
`'&Save Image'`, please. It's most generally an image that is saved. It could be argued that "dot" , the third image exporter, is also an image format. :stuck_out_tongue: